kafka_consumer/check.py

```diff
@@ -227,8 +227,8 @@ def _validate_consumer_groups(self, val):
                                     assert isinstance(partition, int)
             return val
         except Exception as e:
-            self.log.exception(e)
-            raise Exception("""The `consumer_groups` value must be a mapping of mappings, like this:
+            self.log.exception('')
```

Comment:
There's a flake here... you probably want to `except Exception:` in the line above.

I've also been thinking about that long exception message. I think we have some more in the check. I think it clutters the source code a little bit (and we should've probably discussed this in a previous review, so my bad). How would you feel about pointing the users to the proper documentation instead? I don't really feel like logs are the place to include configuration instructions.

If we go that route, I would just re-raise the actual exception after logging something (or not even).